Source/Lib/Trakt.NET/Modules/TraktCommentsModule.cs

```diff
@@ -169,6 +172,41 @@ public async Task<IEnumerable<TraktResponse<ITraktComment>>> GetMutlipleComments
             return comments.ToList();
         }
 
+        /// <summary>
+        /// Gets multiple different <see cref="ITraktComment" />s or replies at once with the given Trakt-Ids or -Slugs.
+        /// <para>OAuth authorization not required.</para>
+        /// <para>
+        /// See <a href="http://docs.trakt.apiary.io/#reference/comments/comment/get-a-comment-or-reply">"Trakt API Doc - Comments: Comment"</a> for more information.
+        /// </para>
+        /// <para>See also <seealso cref="GetCommentAsync(uint, CancellationToken)" />.</para>
+        /// </summary>
+        /// <param name="commentIds">A list of comment ids.</param>
+        /// <param name="cancellationToken">
+        /// Propagates notification that the request should be canceled.<para/>
+        /// If provided, the exception <see cref="OperationCanceledException" /> should be catched.
+        /// </param>
+        /// <returns>An <a href="https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.iasyncenumerable-1?view=net-7.0">async stream</a> of <see cref="ITraktComment" /> instances with the data of each queried comment.</returns>
+        /// <exception cref="TraktException">Thrown, if one request fails.</exception>
+        /// <exception cref="TraktRequestValidationException">Thrown, if validation of request data fails.</exception>
+        public async IAsyncEnumerable<TraktResponse<ITraktComment>> GetCommentsStreamAsync(List<uint> commentIds, [EnumeratorCancellation] CancellationToken cancellationToken = default)
+        {
+            if (commentIds == null || commentIds.Count == 0)
```

Comment:
I'm not an expert with `async yield` and `async break`.
If this happens, what will be the actual result of the method? `null`?

Reply:
Yield break exits out of the function and you have an empty IAsyncEnumerable<T>. It corresponds to return Enumerable.Empty<T> or new List<T>();

Reply:
Ah, okay, thanks. As long as it is not `null`, it's fine.